portfolio.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Site</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <header class="navtools_container">
+      <h2 class="logo">Welcome</h2>
+ 
+      <nav class="navbar">
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </header>
+
+    <main class="main_container">
+      <h1 class="item1">Hey there. I'm Hamza</h1>
+
+      <h1 class="item2">I'm a software developer</h1>
+
+      <P class="item3">
+        I can help you build a product, feature or website 
+        Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.
+      </P>
+
+    <section class="icon_container">
+        <a href="https://www.twitter.com/"><img src="twitter.png" alt="Twitter icon"></a>
+        <a href="https://www.linkedin.com/"><img src="Linkedin icon.png" alt="linkedin icon"></a>
+        <a href="https://medium.com/"><img src="medium.png" alt="medium icon"></a>
+        <a href="https://github.com/"><img src="github.png" alt="github icon"></a>
+        <a href="https://www.angellist.com/"><img src="angellist.png" alt="angellist icon"></a>  
+    </section>
+    </main>
```

Comment:
- [ Optional ] You did a great job by using main but  it is better to use one section for the container of this  section :+1:  it will make your code look clean and more readable